Sources/EdgeNetworkHandlers/EdgeHit.swift

```diff
@@ -25,8 +25,8 @@ protocol EdgeHit {
     /// Unique identifier for this hit
     var requestId: String { get }
```

Comment:
nit: How about renaming the `EdgeHit` to `EdgeRequest`
also renaming several fields
`edgeEndpoint` to `endpoint`
`requestId` to `id`

Reply:
I noticed we have EdgeRequests as well, then should we rename requestId, to just Id Which will translate to hitId. If we have request and hit as two different terms, I feel we can do some cleanup to avoid any confusions.

Reply:
I updated the comment for `requestId` and renamed field edgeEndpoint -> endpoint as the class name contains Edge already, usage example:
```
ExperienceEventsEdgeHit(endpoint: EdgeEndpoint(...), configId: configId, request: requestPayload)
```